datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/glossary/CreateGlossaryTermResolver.java

```diff
@@ -41,9 +43,10 @@ public CompletableFuture<String> get(DataFetchingEnvironment environment) throws
 
     final QueryContext context = environment.getContext();
     final CreateGlossaryEntityInput input = bindArgument(environment.getArgument("input"), CreateGlossaryEntityInput.class);
+    final Urn parentNode = input.getParentNode() != null ? UrnUtils.getUrn(input.getParentNode()) : null;
 
     return CompletableFuture.supplyAsync(() -> {
-      if (AuthorizationUtils.canManageGlossaries((context))) {
+      if (GlossaryUtils.canManageGlossaryEntity(context, parentNode)) {
```

Comment:
Yeah seeing it once more I would expeet to pass the _current_ entity urn into this method

Reply:
yeah these comments make total sense. I think it either needs to be `canManageChildrenEntities` and pass in the `parentNode` or it should still be `canManageGlossaryEntity` and pass in the current entity and an optional parentNode, then the method gets the parent node if none is provided (because sometimes we have access to it like in create while other times we need to fetch it like for delete)

I'm going to go with `canManageChildrenEntities` for now since that's how it's currently set up and I think makes the privileges relationship in the glossary a bit more obvious - we look to owners of the parent for permissions and the name of the method tells us that.

If you like the other option though, let me know! i could be convinced.